doc/rtd/reference/datasources/nocloud.rst

```diff
@@ -21,8 +21,8 @@ Configuration Methods:
 Method 1: Labeled filesystem
 ----------------------------
 
-A labeled `vfat`_ or `iso9660` filesystem containing user data and metadata
-files may be used. The filesystem volume must be labelled ``CIDATA``.
+A labeled `vfat`_ or `iso9660` filesystem may be used. The filesystem volume
```

Comment:
Note that the even disks labeled `CIDATA`  which don't have both `meta-data` and `user-data` files will be ignored. I think that's what the original text was trying to allude to. But, it fell short of that goal. Maybe in the Path resource section below we can mention `The path pointed to by the URI must contain the following files or it will be ignored`